src/install/doinst.sh

```
@@ -1 +1,3 @@
-( cd usr/local/bin ; ln -sf /usr/local/bin/storcli64 storcli )
\ No newline at end of file
+( cd usr/local/bin ; ln -sf /usr/local/bin/storcli64-3404 storcli )
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)